source/user-manual/reference/ossec-conf/client-buffer.rst

```diff
@@ -54,10 +55,21 @@ events_per_second
 
 Specifies the number of events that can be sent to the manager per second.
 
++--------------------+----------------------------------------------------------------------+
+| **Default value**  | 500                                                                  |
++--------------------+----------------------------------------------------------------------+
+| **Allowed values** | Any number between 0 and 100000. Zero to disable the functionality.  |
++--------------------+----------------------------------------------------------------------+
+
+eps_timeframe
+^^^^^^^^^^^^^
+
+Indicates the time interval (in seconds) in which the events per second will be limited.
```

Comment:
This setting is difficult to explain but I think we can improve this description.
```suggestion
Defines the time interval (in seconds) in which the maximum number of events sent is defined by `events_per_second`. 
```